index.html

```diff
@@ -18,6 +18,8 @@ <h1>Awesome Books</h1>
         <button type="submit"> Add </button>
     </form>
 
+    <script src="js/storage.js"></script>
+    <script src="js/Book.js"></script>
     <script src="js/app.js"></script>
```

Comment:
[OPTIONAL]

- Instead of adding all the `js` files in HTML, you can put only one js file. like `index.js` and then you can use all the modules by importing them to the `index.js` file.